docs/alias.connect.md

```diff
@@ -0,0 +1,11 @@
+It's common for browsers to try to prefix domain names with ```www```. Your site doesn't allow for this, so ```www.yoursite.com``` leads to a 404 error page.
+
+# How do I fix this ?
+
+Contact your DNS provider, or log in to the admin console they provide. Then, add a ```CNAME``` record, for ```www``` to point to your root domain.
```

Comment:
If there are a few example and discussions of how to do this using NGINX/Apache as just pointing the CNAME would just serve the same content